ui/src/app/views/new-notebook-modal/component.ts

```diff
@@ -0,0 +1,78 @@
+import {Component, Input, OnDestroy} from '@angular/core';
+
+import {SignInService} from 'app/services/sign-in.service';
+import {Kernels} from 'app/views/notebook-redirect/component';
```

Comment:
This dependency doesn't seem ideal, would prefer pulling this enum out to a shared location, e.g. utils

Reply:
Will do!